src/main/kotlin/util/Constants.kt

```diff
@@ -0,0 +1,16 @@
+package util
+
+class Constants {
+
+    companion object {
```

Comment:
```suggestion
object Constants {
```
클래스 내 companion object만 존재한다면, object를 활용해보면 어떨까요?